### ISSUE TITLE
interesting reference - human pose estimation

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#64]
https://www.youtube.com/watch?v=Dhkd_bAwwMc&feature=youtu.be

'human pose estimation' - I'm assuming this would have required an extensive training set (like marking the limbs with ellipsoids perhaps..) , although i guess you might also have been able to do it from motion capture, (multiple cameras on natural scenes?) or 3d human meshes (render the training set) .. but that's also very data-intensive

This is definitely the sort of thing I think the open-sourced AI community needs additional focus to acheive (and an area of interest between what image-monkey could do and the things i'd like to mess with)

**Comment by bbernhard** [comment]
really interesting, thanks for sharing! 

We already talked a bit about different uses cases here [1], and I think this would be another great use case we could focus on. However, one problem (that you also already mentioned) is: Where do we get the source images/videos from? 

The last few weeks I was looking for people with dash cams (for the "traffic/street" use case). I turned out that it's quite hard to find people who are willing to release their dash cam videos in public domain. Up to now I haven't found anyone...so I am thinking about buying a dash cam myself. 

I think the biggest problem is, that people without technical knowledge are a bit afraid that their data is abused, so they don't want to participate in that. Also the fact that ImageMonkey is still very small and unknown to most people, probably won't make it better ;-). I really hope that we can reach more people in the future and spread the idea of a public, open image dataset. :)

[1] https://github.com/bbernhard/imagemonkey-core/issues/45

**Comment by dobkeratops** [comment]
>Up to now I haven't found anyone...so I am thinking about buying a dash cam myself.

I regret not organising myself with cameras to record bike rides over the years, especially dual cameras. i've recorded a little with a smartphone camera, but a go pro or similar would be more practical

**Comment by bbernhard** [comment]
> I regret not organising myself with cameras to record bike rides over the years, especially dual cameras. i've recorded a little with a smartphone camera with improvised mount, but a go pro or similar would be more practical

yeah, right, bike rides could be really interesting too....I imagine that the quality of the resulting images might be even better. 

My hope is, that if we start with a visual net use case where we can gather enough data ourselves (e.q "traffic/driving" dataset), that other people will follow, as they see that nothing bad happens with their data and that we are using their data in a responsible way. But I might be wrong on that ;-)

**Comment by dobkeratops** [comment]
>  bike rides could be really interesting too

I think a bicycle can better sample all the back streets, pedestrian areas .. the kind of terrain you'll want delivery bots on. I think it would indeed complement dash cam footage in a useful way
